ARCHITECTURE.md

```diff
@@ -41,4 +42,27 @@ For more details about the models we use in Scanner see web-monitoring-db's [API
 - web-monitoring-processing and web-monitoring-versionista-scraper are manually deployed to servers on AWS.
 - We plan to re-deploy these to AWS as a Kubernetes cluster, **but that doesn’t exist yet.**
 
+## Current Workflow
 
+| Diagram key | What happens | What does this | How | Criteria |
+|-|-------------|-|-------|---------|
+| A | Versionista is scraped | [web-monitoring-versionista-scraper](https://github.com/edgi-govdata-archiving/web-monitoring-versionista-scraper) | Data is scraped from Versionista, uploaded to S3, formatted, and [POST](https://api.monitoring.envirodatagov.org/#/imports/post_imports_)ed to [-db](https://github.com/edgi-govdata-archiving/web-monitoring-db) | Runs on a cron job, formats per [GH issue comment](https://github.com/qri-io/walk/issues/16#issuecomment-437785099) |
+|B| IA's Wayback Machine is queried for imports | [web-monitoring-processing](https://github.com/edgi-govdata-archiving/web-monitoring-processing) - this [PR](https://github.com/edgi-govdata-archiving/web-monitoring-processing/pull/174) | Data is pulled from the IA API, formatted, and [POST](https://api.monitoring.envirodatagov.org/#/imports/post_imports_)ed to [-db](https://github.com/edgi-govdata-archiving/web-monitoring-db) | Runs on a cron job, formats per [GH issue comment](https://github.com/qri-io/walk/issues/16#issuecomment-437785099) |
```

Comment:
I think I use IA a lot as shorthand because we say it so much, but in a doc, we should probably write it out as “Internet Archive” or “Wayback” (as appropriate, I guess).

It might also be good to link this [IA API site](https://archive.readme.io/docs) — it’s the best resource as far as I know, but can be surprisingly hard to find.